src/pages/workspace/WorkspaceInvitePage.tsx

```diff
@@ -111,57 +100,69 @@ function WorkspaceInvitePage(props) {
 
     useNetwork({onReconnect: openWorkspaceInvitePage});
 
-    const excludedUsers = useMemo(() => PolicyUtils.getIneligibleInvitees(props.policyMembers, props.personalDetails), [props.policyMembers, props.personalDetails]);
+    const excludedUsers = useMemo(() => PolicyUtils.getIneligibleInvitees(policyMembers, personalDetailsProp), [policyMembers, personalDetailsProp]);
 
     useEffect(() => {
-        const newUsersToInviteDict = {};
-        const newPersonalDetailsDict = {};
-        const newSelectedOptionsDict = {};
+        const newUsersToInviteDict: Record<number, OptionData> = {};
+        const newPersonalDetailsDict: Record<number, OptionData> = {};
+        const newSelectedOptionsDict: Record<number, MemberForList> = {};
 
-        const inviteOptions = OptionsListUtils.getMemberInviteOptions(props.personalDetails, props.betas, searchTerm, excludedUsers, true);
+        const inviteOptions = OptionsListUtils.getMemberInviteOptions(personalDetailsProp, betas ?? [], searchTerm, excludedUsers, true);
 
         // Update selectedOptions with the latest personalDetails and policyMembers information
-        const detailsMap = {};
-        _.each(inviteOptions.personalDetails, (detail) => (detailsMap[detail.login] = OptionsListUtils.formatMemberForList(detail)));
+        const detailsMap: Record<string, MemberForList> = {};
+        inviteOptions.personalDetails.forEach((detail) => {
+            if (!detail.login) {
+                return;
+            }
 
-        const newSelectedOptions = [];
-        _.each(_.keys(props.invitedEmailsToAccountIDsDraft), (login) => {
-            if (!_.has(detailsMap, login)) {
+            detailsMap[detail.login] = OptionsListUtils.formatMemberForList(detail);
+        });
+
+        const newSelectedOptions: MemberForList[] = [];
+        Object.keys(invitedEmailsToAccountIDsDraft ?? {}).forEach((login) => {
+            if (!(login in detailsMap)) {
                 return;
             }
             newSelectedOptions.push({...detailsMap[login], isSelected: true});
         });
-        _.each(selectedOptions, (option) => {
-            newSelectedOptions.push(_.has(detailsMap, option.login) ? {...detailsMap[option.login], isSelected: true} : option);
+        selectedOptions.forEach((option) => {
+            newSelectedOptions.push(option.login && option.login in detailsMap ? {...detailsMap[option.login], isSelected: true} : option);
         });
 
         const userToInvite = inviteOptions.userToInvite;
 
         // Only add the user to the invites list if it is valid
-        if (userToInvite) {
+        if (typeof userToInvite?.accountID === 'number') {
```

Comment:
```suggestion
        if (userToInvite?.accountID) {
```

Couldn't we just do this?

Reply:
In theory it's possible `accountID` will be 0 and the check won't work as expected